### PR TITLE
[MH2018-21] ->  [MH-Team-1-Shopify]

### PR DESCRIPTION
**Pull request by rubiocanino**
Just adding the main photo and the thumbnails to the PDP.

Original:
image

How looks:
image

Original:
image

How looks:
screenshot from 2018-04-02 16-19-47

### CODE REVIEW

File: fiorella/src/styles/pages/_product.scss [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+        margin-top: 0;
+      }
+
+      @media #{ $small-desktop-view } {
+        height: auto;
+        max-width: 100%;
+      }
+    }
+
+    .view {
```
Comment (magmahound) [inline fiorella/src/styles/pages/_product.scss]: Selector should have depth of applicability no greater than 2, but was 3

File: fiorella/src/styles/pages/_product.scss [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+            &:hover {
+              border-color: $border-color-white;
+              border-width: 2px;
+              opacity: 0.5;
+            }
+          }
+        }
+      }
+
+      &:last-child {
```
Comment (magmahound) [inline fiorella/src/styles/pages/_product.scss]: Selector should have depth of applicability no greater than 2, but was 3

File: fiorella/src/styles/pages/_product.scss [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+          &:hover {
+            background-color: $color-background-thumbnail;
+            border: 2px solid $border-color-white;
+          }
+
+          img {
+            object-fit: contain;
+            position: relative;
+
+            &:hover {
```
Comment (magmahound) [inline fiorella/src/styles/pages/_product.scss]: Selector should have depth of applicability no greater than 2, but was 6

File: fiorella/src/styles/pages/_product.scss [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+        .thumbnail-hover {
+          height: 70px;
+          width: 70px;
+
+          &:hover {
+            background-color: $color-background-thumbnail;
+            border: 2px solid $border-color-white;
+          }
+
+          img {
```
Comment (magmahound) [inline fiorella/src/styles/pages/_product.scss]: Selector should have depth of applicability no greater than 2, but was 6

File: fiorella/src/styles/pages/_product.scss [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+          border: 1px solid $color-hover;
+          height: 72px;
+          width: 72px;
+        }
+
+        .thumbnail-hover {
+          height: 70px;
+          width: 70px;
+
+          &:hover {
```
Comment (magmahound) [inline fiorella/src/styles/pages/_product.scss]: Selector should have depth of applicability no greater than 2, but was 5

File: fiorella/src/styles/pages/_product.scss [hunk 6]
```diff
@@ -0,0 +1,115 @@
...
+      display: flex;
+    }
+
+    a {
+      height: 85px;
+      margin-right: 20px;
+      padding-top: 10px;
+      width: 95%;
+
+      .thumbnail-pink {
```
Comment (magmahound) [inline fiorella/src/styles/pages/_product.scss]: Selector should have depth of applicability no greater than 2, but was 4

File: fiorella/src/styles/pages/_product.scss [hunk 7]
```diff
@@ -0,0 +1,115 @@
...
+    justify-content: center;
+    padding-left: 20px;
+    padding-top: 25px;
+    width: 70%;
+
+    @media #{ $small-desktop-view } {
+      display: flex;
+    }
+
+    a {
```
Comment (magmahound) [inline fiorella/src/styles/pages/_product.scss]: Selector should have depth of applicability no greater than 2, but was 3

File: fiorella/src/styles/pages/_product.scss [hunk 8]
```diff
@@ -0,0 +1,115 @@
...
+    width: 50%;
+  }
+
+  .box-product {
+    display: flex;
+    height: 400px;
+    justify-content: center;
+    width: 100%;
+
+    .product {
```
Comment (magmahound) [inline fiorella/src/styles/pages/_product.scss]: Selector should have depth of applicability no greater than 2, but was 3

File: fiorella/src/styles/pages/_product.scss [hunk 9]
```diff
@@ -0,0 +1,127 @@
+.title{
```
Comment (magmahound) [inline fiorella/src/styles/pages/_product.scss]: Opening curly brace `{` should be preceded by one space

File: fiorella/src/styles/pages/_product.scss [hunk 10]
```diff
@@ -0,0 +1,127 @@
...
+    padding: 0;
```
Comment (IsmaelVillavicencio) [inline fiorella/src/styles/pages/_product.scss]: Please reorder the properties in alphabetical order

File: fiorella/src/styles/pages/_product.scss [hunk 11]
```diff
@@ -0,0 +1,127 @@
...
+    margin: 0;
+    text-align: center;
+    font-size: 28px;
+    font-family: $merriweathersans;
+    
+    @media #{ $small-desktop-view }{
```
Comment (jaimelr) [inline fiorella/src/styles/pages/_product.scss]: Opening curly brace `{` should be preceded by one space

File: fiorella/src/styles/abstracts/_variables.scss.liquid [hunk 12]
```diff
@@ -7,6 +7,8 @@ $color-body-text: {{ settings.color_body_text }};
 
 // Backgrounds
 $color-body: #fff;
+$color-hover: #C0392B;
+$color-background-thumbnail: rgba(255,93,177,1);
```
Comment (jaimelr) [inline fiorella/src/styles/abstracts/_variables.scss.liquid]: There are some missing spaces after the `,`

File: fiorella/src/styles/pages/_product.scss [hunk 13]
```diff
@@ -0,0 +1,127 @@
...
+      height: 85px;
+      margin-right: 20px;
+      padding-top: 10px;
+      width: 95%;
+
+      .thumbnail-pink {
+        height: 75px;
+        width: 75px;
+
+        &:hover {
```
Comment (magmahound) [inline fiorella/src/styles/pages/_product.scss]: Nesting should be no greater than 4, but was 5<br>Selector should have depth of applicability no greater than 2, but was 4

**Comment by rubiocanino** [comment]
@jaimelr  The deep level 6 it's for the hover on the image, actually isn't another level just a action that the image had

File: fiorella/src/styles/pages/_product.scss [hunk 14]
```diff
@@ -0,0 +1,124 @@
...
+      justify-content: flex-start;
+      padding-top: 0;
+      width: 150px;
+    }
+  }
+
+  .view {
+    height: 85px;
+    object-fit: cover;
+    width: 85px;
```
Comment (magmahound) [inline fiorella/src/styles/pages/_product.scss]: Properties should be ordered height, object-fit, object-fit, position, width

File: fiorella/src/styles/pages/_product.scss [hunk 15]
```diff
@@ -0,0 +1,124 @@
...
+     height: 70px;
+     width: 70px;
+
+     &:hover {
+       background-color: $color-background-thumbnail;
+       border: 2px solid $border-color-white;
+     }
+   }
+
+   .thumbnail-pink {
```
Comment (magmahound) [inline fiorella/src/styles/pages/_product.scss]: Line should be indented 0 spaces, but was indented 3 spaces

File: fiorella/src/styles/pages/_product.scss [hunk 16]
```diff
@@ -0,0 +1,124 @@
...
+    }
+
+    @media #{ $small-desktop-view } {
+      height: 70px;
+      object-fit: cover;
+      width: 70px;
+    }
+  }
+
+  .thumbnail-hover {
```
Comment (magmahound) [inline fiorella/src/styles/pages/_product.scss]: Line should be indented 0 spaces, but was indented 2 spaces

File: fiorella/src/styles/pages/_product.scss [hunk 17]
```diff
@@ -0,0 +1,124 @@
...
+      align-self: center;
+      flex-direction: column;
+      height: 100%;
+      justify-content: flex-start;
+      padding-top: 0;
+      width: 150px;
+    }
+  }
+
+  .view {
```
Comment (magmahound) [inline fiorella/src/styles/pages/_product.scss]: Line should be indented 0 spaces, but was indented 2 spaces

File: fiorella/src/styles/pages/_product.scss [hunk 18]
```diff
@@ -0,0 +1,124 @@
...
+      width: 100%;
+    }
+
+    @media #{ $small-desktop-view } {
+      width: 100%;
+    }
+  }
+}
+
+  .box-micro {
```
Comment (magmahound) [inline fiorella/src/styles/pages/_product.scss]: Line should be indented 0 spaces, but was indented 2 spaces

File: fiorella/src/styles/pages/_product.scss [hunk 19]
```diff
@@ -0,0 +1,124 @@
+.title {
+    font-family: $merriweathersans;
+    font-size: 28px;
+    margin: 0;
+    padding: 0;
+    text-align: center;
+
+    @media #{ $small-desktop-view } {
```
Comment (magmahound) [inline fiorella/src/styles/pages/_product.scss]: Line should be indented 2 spaces, but was indented 4 spaces

File: fiorella/src/styles/pages/_product.scss [hunk 20]
```diff
@@ -0,0 +1,124 @@
+.title {
+    font-family: $merriweathersans;
+    font-size: 28px;
+    margin: 0;
+    padding: 0;
+    text-align: center;
```
Comment (magmahound) [inline fiorella/src/styles/pages/_product.scss]: Line should be indented 2 spaces, but was indented 4 spaces

File: fiorella/src/styles/pages/_product.scss [hunk 21]
```diff
@@ -0,0 +1,124 @@
+.title {
+    font-family: $merriweathersans;
+    font-size: 28px;
+    margin: 0;
+    padding: 0;
```
Comment (magmahound) [inline fiorella/src/styles/pages/_product.scss]: Line should be indented 2 spaces, but was indented 4 spaces

File: fiorella/src/styles/pages/_product.scss [hunk 22]
```diff
@@ -0,0 +1,124 @@
+.title {
+    font-family: $merriweathersans;
+    font-size: 28px;
+    margin: 0;
```
Comment (magmahound) [inline fiorella/src/styles/pages/_product.scss]: Line should be indented 2 spaces, but was indented 4 spaces

File: fiorella/src/styles/pages/_product.scss [hunk 23]
```diff
@@ -0,0 +1,124 @@
...
+      padding-top: 0;
+      width: 150px;
+    }
+  }
+
+  .view {
+    height: 85px;
+    object-fit: cover;
+    width: 85px;
+    object-fit: contain;
```
Comment (magmahound) [inline fiorella/src/styles/pages/_product.scss]: Property `object-fit` already defined on line 87

File: fiorella/src/scripts/theme.js [hunk 24]
```diff
@@ -52,3 +52,7 @@ $(document).ready(function() {
     document.documentElement.className = document.documentElement.className.replace('supports-no-cookies', 'supports-cookies');
   }
 });
+
+function selectimage(image) {
```
Comment (magmahound) [inline fiorella/src/scripts/theme.js]: 'selectimage' is defined but never used.

File: fiorella/src/styles/pages/_product.scss [hunk 25]
```diff
@@ -0,0 +1,124 @@
...
+  }
+
+  .box-product {
+    display: flex;
+    height: 400px;
+    justify-content: center;
+    width: 100%;
+
+    .product {
+      object-fit: contain;
```
Comment (magmahound) [inline fiorella/src/styles/pages/_product.scss]: Properties should be ordered height, justify-content, object-fit, width

File: fiorella/src/styles/pages/_product.scss [hunk 26]
```diff
@@ -0,0 +1,123 @@
...
+      flex-direction: column;
+      height: 100%;
+      justify-content: flex-start;
+      padding-top: 0;
+      width: 150px;
+    }
+  }
+
+  .view {
+    object-fit: contain;
```
Comment (magmahound) [inline fiorella/src/styles/pages/_product.scss]: Properties should be ordered height, object-fit, position, width

File: fiorella/src/styles/pages/_product.scss [hunk 27]
```diff
@@ -0,0 +1,123 @@
...
+      height: 100%;
+      justify-content: flex-start;
+      padding-top: 0;
+      width: 150px;
+    }
+  }
+
+  .view {
+    height: 85px;
+    width: 85px;
```
Comment (magmahound) [inline fiorella/src/styles/pages/_product.scss]: Properties should be ordered height, object-fit, position, width

**Pull request merged**